runhouse/rns/rns_client.py

```diff
@@ -551,7 +551,7 @@ def delete_configs(
 
         if rns_address.startswith("/"):
             resource_uri = self.resource_uri(rns_address)
-            uri = f"{self.api_server_url}/resource/{resource_uri}"
+            uri = f"{self.api_server_url}/resource/{resource_uri}?delete_creds=false"
```

Comment:
defaults to True in rh-den currently to retain current behavior, but we can remove the flag at release and handle this logic purely in `cluster.delete_configs` instead of in den